### ISSUE TITLE
Lock Up

### ISSUE THREAD

**Issue by Thack637** [Courseplay/courseplay#6705]
[crashlog.zip](https://github.com/Courseplay/courseplay/files/5853606/crashlog.zip)
Hi Guys,
I am having lockups. I have traced it to courseplay not getting along well with another mod or mods. I did this by uninstalling courseplay and the lock ups quit immediately. I have attached a copy of my log.txt file so hopefully you can see what is causing the issue. I hope that you guys are able to see what is causing the issue as I love your mod and do not want to quit using it. The lock ups occur when I am trying to switch vehicles either with the tab key or by opening the map and clicking on the vehicle in the map. I can usually play for about 30-60 mins before it happens, sometimes less.

**Comment by Tensuko** [comment]
Not sure but you might want remove this:
(Version: 1.0.0.0) FS19_AutoRepair
We have a build in AutoRepair in the advanced settings screen.

That might be the JD 7R ( I remember we had Issues with this before already )
2021-01-21 19:36 Error: Can't load resource 'D:/Users/Rob/Documents/My Games/FarmingSimulator2019/mods/FS19_JohnDeere_7R/scripts/AddConfig.lua'.

Then there is this Callstack:
2021-01-21 20:37 Warning (script): Unknown entity id 909718 in method 'getParent'.
2021-01-21 20:37 LUA call stack:
2021-01-21 20:37   dataS/scripts/vehicles/Vehicle.lua (1703) : getParent
2021-01-21 20:37   dataS/scripts/vehicles/specializations/Suspensions.lua (259) : getParentComponent
2021-01-21 20:37   dataS/scripts/vehicles/specializations/Suspensions.lua (276) : setSuspensionNodeCharacter
2021-01-21 20:37   dataS/scripts/vehicles/SpecializationUtil.lua (30)
2021-01-21 20:37   dataS/scripts/vehicles/specializations/Enterable.lua (619) : raiseEvent
2021-01-21 20:37   dataS/scripts/BaseMission.lua (1881) : enterVehicle
2021-01-21 20:37   dataS/scripts/utils/Utils.lua (393) : oldFunc
2021-01-21 20:37   dataS/scripts/FSBaseMission.lua (3245) : onEnterVehicle
2021-01-21 20:37   dataS/scripts/vehicles/VehicleEnterResponseEvent.lua (66) : onEnterVehicle
2021-01-21 20:37   dataS/scripts/network/Connection.lua (77) : run
2021-01-21 20:37   dataS/scripts/vehicles/VehicleEnterRequestEvent.lua (68) : sendEvent
2021-01-21 20:37   dataS/scripts/network/Connection.lua (77) : run
2021-01-21 20:37   dataS/scripts/BaseMission.lua (1852) : sendEvent
2021-01-21 20:37   dataS/scripts/gui/InGameMenuMapFrame.lua (1370) : requestToEnterVehicle
2021-01-21 20:37   dataS/scripts/gui/InGameMenuMapFrame.lua (1579) : onClickEnterVehicle
2021-01-21 20:37   dataS/scripts/input/InputEvent.lua (100) : callback
2021-01-21 20:37   dataS/scripts/input/InputBinding.lua (3219) : notifyInput
2021-01-21 20:37   dataS/scripts/input/InputBinding.lua (2937) : updateInput
2021-01-21 20:37   dataS/scripts/main.lua (1916) : update
2021-01-21 20:37 Warning (script): 'getParent': Argument 1 has wrong type. Expected: Int. Actual: Nil 
2021-01-21 20:37   ..\..\..\src\base\scripting\lua\LuaScriptSystem.cpp (882): expectedType == Value::VoidType || expectedType == Value::ClassType

**Comment by pvaiko** [comment]
As @Tensuko said, this is a problem with one of your vehicles, most likely that JD. If you look at the error messages you see they are not even from Courseplay, those from the standard game scripts.

**Comment by Thack637** [comment]
HI Guys, 
Thank you so much for taking the time to look at the issues I am having. I am not a programmer and I am not sure what I am looking at when I look at things like the log.txt I uploaded when trying to identify issues.  It is in fact the one that came from FS19. Is there a log file similar to that for courseplay that I can look at to try to identify any further issues I may have? I will remove the items you suggested and see if that resolves the issue.  Thanks again for your help.

**Comment by Tensuko** [comment]
The log.txt you uploaded here is the one from FS19.
The game and also some mods do print debuginformations and Errors in this logfile.
Thats why it is so important to post it too and thats how we see the LUA Error of the FS19_JohnDeere_7R.

**Comment by Thack637** [comment]
Hi Tensuko,
I have removed anything that had a JD 7r in it. I have also removed the auto repair mod that you pinpointed. I am still getting the crash with the LUA error in it. I started a new game that I left all of the actual vehicle mods out and I had another crash last night. I have included the log.txt file with this from FS19. Is there one for CP that I can look at?
[log.zip](https://github.com/Courseplay/courseplay/files/5860679/log.zip)
